WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -91,7 +91,7 @@ public static bool PossiblyP2WPKHInvolved(this Transaction me)
 		}
 		foreach (TxOut output in me.Outputs)
 		{
-			if (output.ScriptPubKey.IsScriptType(ScriptType.P2WPKH))
+			if (output.ScriptPubKey.IsScriptType(ScriptType.P2WPKH) || output.ScriptPubKey.IsScriptType(ScriptType.Taproot))
```

Comment:
Yes, the thing is that that method is wrong in more than one way. It was supposed to detect segwit transactions but even a unsigned non-segwit transaction returns `true`. I think it should use `Transaction.HasWitness` on the inputs checks and be called `SegwitInvolved`. 

I can rename it but fixing it would involve even more testing and review because it changes the transaction processing code.

Reply:
Renamed.